packages/web-config-server/src/apiV1/dataBuilders/modules/disaster/disasterAffectedOrganisationOperationalData.js

```diff
@@ -16,7 +16,7 @@ export const disasterAffectedOrganisationOperationalData = async (
 
   if (!disasterStartDate) return { data: [] };
 
-  const facilities = Entity.getFacilitiesOfOrgUnit(organisationUnitCode);
+  const facilities = await Entity.getFacilitiesOfOrgUnit(organisationUnitCode);
```

Comment:
Tagging myself as this will be a conflict with the entity hierarchy stuff @edmofro

Reply:
Resolved in https://github.com/beyondessential/tupaia/pull/1484/commits/78da7b8fe4d98165f6a82fbafdc5a9902c5e577e (accepted my version, where I'd also added an await)